nss-cohort-37/Bangazon-Site-Silver-Snakes
# Jan productsell

# Description

Users can add a product, MUST select a category, and then are directed to the details page

Fixes #1 #10 #13 

## Type of change

Please delete options that are not relevant.


# Testing Instructions
DO THIS TWICE
1. Select sell a product
2. Enter in the requisite information **leaving off the category**
3. confirm the alert window appears and your previous info also remains
4. select a category and create it
5. make sure you're on the detail page with the information you just added visible

# Checklist:
